boards/arm/holyiot_yj16019/Kconfig.board

```diff
@@ -0,0 +1,9 @@
+# Kconfig - Holyiot YJ-16019 board configuration
+#
+# Copyright (c) 2019 Henrik Brix Andersen <henrik@brixandersen.dk>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_HOLYIOT_YJ16019
+	bool "Holyiot YJ-16019"
+	depends on SOC_NRF52832_QFAA
```

Comment:
Can this be a `select` instead of `depends` on?

Reply:
I guess so, but it looks like all the other nRF5x boards are using `depends`?

Reply:
See if `select` works, and if you don't get any odd build warnings.

If it does, we should be using `select` as it makes more sense than `depends`.

Reply:
```warning: the choice symbol SOC_NRF52832_QFAA (defined at soc/arm/nordic_nrf/nrf52/Kconfig.soc:197) is selected by the following symbols, but select/imply has no effect on choice symbols```

So I guess the other boards are using `depends` for this reason.

Reply:
Yeah, I wasn't sure, so stick with `depends` :)